tests/unittests/Foundation/NSOperationTests.mm

```diff
@@ -88,14 +88,15 @@ - (instancetype)initWithBlock:(void (^)())block;
 @implementation BlockThread
 - (instancetype)initWithBlock:(void (^)())block {
     if (self = [super init]) {
-        self->_block = block;
-        self->_finishCondition = [[_NSBooleanCondition new] autorelease];
+        self.block = block;
```

Comment:
though, wait; we generally avoid using setters in init, because they can be overridden by subclasses who are not yet initialized.